.azure-pipelines/test_plan.py

```diff
@@ -823,8 +823,9 @@ def poll(self, test_plan_id, interval=60, timeout=-1, expected_state="", expecte
 
             scripts = args.scripts
             specific_param = []
-            # For KVM PR test, get test modules from pr_test_scripts.yaml, otherwise use args.scripts
-            if args.platform == "kvm":
+            # For kvm platform, if args.features or args.scripts is not None, get test modules form it.
+            # Otherwise, get test modules from pr_test_scripts.yaml.
+            if args.platform == "kvm" and (not args.features and not args.scripts):
```

Comment:
Suggest:
`explicitly_specify_test_module = args.features or args.scripts`
`if args.platform == "kvm" and (not explicitly_specy_test_module) :`

Reply:
Can we use `test plan type == "PR"` to instead `args.platform == "kvm"` ?

Reply:
> Suggest: `explicitly_specify_test_module = args.features or args.scripts` `if args.platform == "kvm" and (not explicitly_specy_test_module) :`

Good suggestion! Thanks.

Reply:
> Can we use `test plan type == "PR"` to instead `args.platform == "kvm"` ?

Yes, please make sure that in practice, these two are equivalent.

Reply:
> Can we use `test plan type == "PR"` to instead `args.platform == "kvm"` ?

Oh, but we don't pass this arg "test_plan_type" in every PR test job.

Reply:
Yeah, they're equivalent, and even though we didn't explicitly pass test_plan_type, by default the test_plan.py set it as PR, you can look into the definition of test_plan_type